src/UserGuide/V1.2.x/User-Manual/Tiered-Storage_timecho.md

```diff
@@ -19,6 +19,78 @@
 
 -->
 
-# Tiered Storage
+# Tiered Storage 
+## Overview
 
-TODO
\ No newline at end of file
+Tiered storage function provides users with the ability to manage tiered storage media. users can use the tiered storage function to configure different types of storage media for IoTDB and to classify the storage media. ioTDB can support tiered storage from memory, SSD, normal hard disc to network hard disc by parameter configuration only according to the degree of hot and cold data. Specifically, in IoTDB, the configuration of tiered storage is reflected in the management of multiple directories. Users can group tiered storage directories into the same category and configure them into IoTDB as a "tier", which is called storage tier; at the same time, users can categorize data according to hot or cold, and store different categories of data into designated storage tiers. Meanwhile, users can categorise data according to hot or cold and store different categories of data in the specified tier. Currently, IoTDB supports the classification of hot and cold data by TTL, when the data in one tier does not meet the TTL rules defined in the current tier, the data will be automatically migrated to the next tier.
+
+## Parameter Definition
+
+To enable tiered storage in IoTDB, you need to configure the following aspects:
+
+1. configure the data catalogue and divide the data catalogue into different tiers
+2. configure the TTL of the data managed in each tier to distinguish between hot and cold data categories managed in different tiers.
+3. configure the minimum remaining storage space ratio for each tier so that when the storage space of the tier triggers the threshold, the data of the tier will be automatically migrated to the next tier (optional).
+
+The specific parameter definitions and their descriptions are as follows.
+
+| Configuration                                    | Default                    | Description                                                          | Constraint                                                          |
+| ---------------------------------------- | ------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| dn_data_dirs                             | None                        | specify different storage directories and divide the storage directories into tiers             | Each level of storage uses a semicolon to separate, and commas to separate within a single level; cloud configuration can only be used as the last level of storage and the first level can't be used as cloud storage; a cloud object at most; the remote storage directory is denoted by OBJECT_STORAGE |
```

Comment:
| dn_data_dirs                             | None                        | specify different storage directories and divide the storage directories into tiers             | Each level of storage uses a semicolon to separate, and commas to separate within a single level; cloud (`OBJECT_STORAGE`) configuration can only be used as the last level of storage and the first level can't be used as cloud storage; a cloud object at most; the remote storage directory is denoted by OBJECT_STORAGE |